app/views/smart_proxies/plugins/_dhcp_subnets_card.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="col-md-3">
+  <div class="card-pf card-pf-aggregate-status card-pf-accented">
+    <%= render :partial => 'smart_proxies/plugins/dhcp_subnets_card_header', :locals => { :subnet => subnet } %>
```

Comment:
no need to split this into another partial just for 3 lines that aren't reused anywhere. Also, should link to the relevant subnet.